src/wechaty_plugin_contrib/contrib/chat_history_plugin/plugin.py

```diff
@@ -18,13 +25,30 @@
     MessageType.MESSAGE_TYPE_AUDIO
 ]
 
+Base = declarative_base()
+
+
+class ChatHistory(Base):
+    """ChatHistory"""
+    __tablename__ = 'ChatHistory'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    msg_id = Column(Integer, default=None)
+    msg_date = Column(DateTime, default=None)
+    msg_type = Column(Integer, default=None)
+    msg_room = Column(VARCHAR(50), default=None)
+    msg_talker = Column(VARCHAR(50), default=None)
+    msg_receiver = Column(VARCHAR(50), default=None)
+    msg_text = Column(Text, default=None)
```

Comment:
If you can keep the field consist with `MessagePayload`, it will significantly reduced conceptual cost about db.  How do you think about it ?

Reply:
- As we know, the field in `MessagePayload` like `from_id`, `to_id`, `room_id` will be dynamic in UOS puppet, so we had better save both the `contact_id` and `identity` (alias name) so that we won't be confused about the name.
- Will add field `filename` to make message more clear.

Reply:
Yes, different puppet protocols have different identified field, so you can set it dynamic. It's simple problem for developer, so you should not care about it.